setup.cfg

```diff
@@ -7,4 +7,5 @@ exclude = mypy/codec/*,mypy/test/data/lib-stub/*,mypy/test/data/fixtures/*
 #   F401: unused identifiers (useless, as it doesn't see inside # type: comments)
 #   W601: has_key() deprecated (false positives)
 #   E701: multiple statements on one line (colon) (we use this for classes with empty body)
-ignore = E251,E128,F401,W601,E701
+#   F403: 'from foo import *' used; unable to detect undefined names (mypy checks this better)
```

Comment:
It's not only because of flake8, it's also for human readers. Anyway changing our style guidelines hidden in a PR is not the right way to get this changed. Feel free to open an issue for this specific purpose (but don't just send a PR as PRs are for things that aren't controversial). It's unlikely to go through though, as consistency with PEP 8 is usually more important some slight convenience.